snow/init.lua

```diff
@@ -0,0 +1,22 @@
+ctf_match.register_on_build_time_start(function()
```

Comment:
`ctf_match.register_on_new_match` would be better here. Why snow only after buildtime?

Reply:
It was due to my original attempts to find the map bounds, I was using the flag positions. Can change that now though...

Reply:
There's the issue of the map using the `on_new_match` too, so there's the possibility of a race condition. But as this mod depends on `ctf_map`, the latter will be executed first, and would subsequently register its `on_new_match` first.

Reply:
Changed